docs/pages/kubernetes-access/helm/guides/gcp.mdx

```diff
@@ -0,0 +1,422 @@
+---
+title: Running an HA Teleport cluster using GCP, GKE and Helm
+description: Install and configure an HA Teleport cluster using a Google Cloud GKE cluster
+---
+
+In this guide, we'll go through how to set up a highly-available Teleport cluster with multiple replicas in Kubernetes
+using Teleport Helm charts and Google Cloud Platform products (Firestore and Google Cloud Storage).
+
+### Install Helm
```

Comment:
Same comments on the structure as in aws - prerequisites, next steps at the bottom.